packages/studio-ui/src/web/components/Layout/TopNav/cloud/workspace.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Intent, MenuItem } from '@blueprintjs/core'
+import { Select } from '@blueprintjs/select'
+import { lang } from 'botpress/shared'
+import React, { useState } from 'react'
+import style from './style.scss'
+import { Workspace } from './types'
+
+const WorkspaceSelect = Select.ofType<Workspace>()
+
+export function WorkspaceSelector(props: {
```

Comment:
we have [this wrapper component](https://github.com/botpress/studio/blob/4ded33437ef9f5f6fe2274dde3f30f56e6c1df39/packages/ui-shared/src/Dropdown/index.tsx#L33-L34) that handles most of the blueprint select boilerplate if that could be of any use

Reply:
I would rather not use it, as it's not type safe: https://github.com/botpress/studio/blob/e5bd786a809ffe3cf9c4927022a7be7a3871a975/packages/ui-shared/src/Dropdown/typings.d.ts#L6